tutorials-v5/time-evolution/011_floquet_solver.md

```diff
@@ -87,7 +84,7 @@ where $\gamma$ is the dissipation rate. The system-bath interaction is described
 
 ```python
 # Noise Spectral Density
-gamma = 0.1
+gamma = 0.5
 
 
 def noise_spectrum(omega):
```

Comment:
We need to add the `(omega >0)` here.

Reply:
Thanks. Added with a short explanation of why this is needed.